frontend/app/vite.config.ts

```diff
@@ -41,7 +41,8 @@ export default defineConfig({
             vue: 'vue/dist/vue.runtime.mjs'
           }
         : {})
-    }
+    },
+    dedupe: ['vue']
```

Comment:
I believe in previous version, this was auto deduped, but in newer version, it's not